lib/engine/game/g_1893/game.rb

```diff
@@ -828,16 +883,22 @@ def operating_round(round_num)
             Engine::Step::Bankrupt,
             Engine::Step::DiscardTrain,
             Engine::Step::HomeToken,
-            Engine::Step::Track,
+            G1893::Step::Track,
             Engine::Step::Token,
             Engine::Step::Route,
             G1893::Step::Dividend,
             G1893::Step::BuyTrain,
           ], round_num: round_num)
         end
 
+        def operating_order
+          # AdSK is bonds, and do not operate
+          super.reject { |c| c == adsk }
```

Comment:
then maybe adsk shouldn't be a corp?

Reply:
Maybe not, but making it a company is not a perfect fit, either. 

Probably there should be a bond class (or similar).

But I am trying it as a company in the latest version.